components/CodeComponent.js

```diff
@@ -0,0 +1,16 @@
+export default function CodeComponent({ code = '' }) {
```

Comment:
Note for later: There should be a way to pass the language of the snippet.

Reply:
Using Shikijs for formatting https://codesandbox.io/s/shiki-next-js-cir0y?file=/pages/index.js